2024.10.31/01_reservations/r1-alex.js

```diff
@@ -0,0 +1,14 @@
+let id = 0;
```

Comment:
why you need this ?

I said `id is index+1`

Reply:
fixed